Projects/My_website/myWebsite/blog/views.py

```diff
@@ -18,3 +19,11 @@ class crudBlogApi(viewsets.ModelViewSet):
     serializer_class = blogSerializer
     authentication_classes = [TokenAuthentication]
     permission_classes = [IsAuthenticated]
+
+    def get(self,request):
+        userid = request.user.id
+        datalist = list(self.queryset)
+        serializers = blogSerializer(datalist,many=True)
+        return Response(serializers.data)
+    
```

Comment:
@codal-uchoudhary 

(i) How will this code filter out user specific Blogs ?
(ii) you're fetching userid, but not using it anywhere.
(iii) What is the need of converting queryset to list ?

Reply:
yes , i am actually working on it. i try lot's of approaches  to fetch user specific data. I believe i will found optimal solution very soon.